README.md

```diff
@@ -1,2 +1,156 @@
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **💱 Currency Showcase**
+- **📊 Population Insights**
+- **🎓 Educational and Informative**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://sonikak004.github.io/react-redux-capstone-project)
+- [Presentation Video Link](https://www.loom.com/share/ab5fe699ec4049c9a3e8aee6cd444806?sid=db630c59-c401-4992-99aa-53520e422575)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- Clone this project.
+- Open it in VS code.
+- Run `npm install` to install the dependencies
+- Run `npm run build` to build the project
+- Run `npm start` to open it in the dev server
+
+### Prerequisites
+
+In order to run this project you need:
+
+Git
+A GitHub account
+VS-Code
+<a href="https://github.com/microverseinc/linters-config/tree/master/html-css-js">HTML, CSS, and JavaScript linters</a>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+https://github.com/Sonikak004/react-redux-capstone-project.git
+
+### Install
+
+Install this project with:
+
+This project doesn't need to be installed
+
+### Usage
+
+Set your wanderlust free and deepen your understanding of our world's nations
+
+### Run tests
+
+To run tests, run the following command: `npm test`
+
+### Deployment
+
+You can deploy this project by using VS-code and injecting it to a browser through a local live server.
+
+Or you can use the dev server. To initiate it, run `npm build run` in your cli and the project will open a dev server at http://localhost:8080/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Sonika**
+
+- GitHub: [@sonikak004](https://github.com/sonikak004)
+- Twitter: [@sonikak004](https://twitter.com/sonikak004)
+- LinkedIn: [sonikak004](https://linkedin.com/in/sonikak004)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Interactive map integration for visual exploration**
+- [ ] **Advanced filtering options, allowing for intricate searches based on various criteria**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Sonikak004/react-redux-capstone-project/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please give it a star ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank <a href="https://www.microverse.org/">Microverse</a> for providing the guidelines that made this project possible.
```

Comment:
- [ ] As stated in the project requirement Kindly make sure that The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).